pandas/core/internals/managers.py

```diff
@@ -623,7 +623,7 @@ def comp(s, regex=False):
         bm._consolidate_inplace()
         return bm
 
-    def is_consolidated(self):
+    def is_consolidated(self) -> bool:
```

Comment:
as an aside, could this be a property? and remove/inline _consolidate_check

Reply:
probably could, would need to do some profiling.  xref #32261.